src/plugins/multimedia/multimedia-fr.hbs

```diff
@@ -15,7 +15,7 @@
 <div class="row">
 	<section class="col-md-12">
 		<h2>Vue d'ensemble</h2>
-		<p>Pour la vidéo, le lecteur multimédia s'appuie sur la balise «&#160;<code>video</code>&#160;» HTML5 natif et s'appuie sur Flash comme un mécanisme de secours lorsque la balise «&#160;<code>video</code>&#160;» HTML5 n'est pas implémentée. Le format MPEG-4 (codec H264) est le minimum requis pour la vidéo parce que le repli flash s'appuie sur elle et H264 est le format vidéo standard sur de nombreux appareils mobiles. Un WebM facultative mais fortement recommandée (VP8 codec) devrait être ajouté aussi bien pour permettre à certains navigateurs tels que Firefox qui ne comprennent pas un support natif pour H264 de tirer parti des gains de performance HTML5 indigènes.</p>
+		<p>Pour la vidéo, le lecteur multimédia s'appuie sur la balise «&#160;<code>video</code>&#160;» HTML5 natif. Le format MPEG-4 (codec H264) est le minimum requis pour la vidéo parce que c'est le format vidéo standard sur de nombreux navigateurs. Un WebM facultative mais fortement recommandée (codec VP8) devrait être ajouté aussi bien pour permettre à certains navigateurs tels que Firefox qui ne comprennent pas un support natif pour H264 de tirer parti des gains de performance HTML5 indigènes.</p>
```

Comment:
"indigènes" are not the right word to use in this context, it's means "indigenous people".

May be use instead: "des gains de performance de leur implémentations d'HTML5"

In fact the full sentence needs to be re-written in French but at least apply my suggestion it will be less bad.